src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -45,6 +45,18 @@ public int BringUpOnIdentifier
             set { SetBooleanOption(CompletionOptions.TriggerOnTypingLetters, value); }
         }
 
+        public int HighlightMatchingPortionsOfCompletionListItems
```

Comment:
We should add integration tests support for these new options to roslyn-internal too.